heartbeat/galera

```diff
@@ -243,12 +243,23 @@ Cluster check user.
 
 <parameter name="check_passwd" unique="0" required="0">
 <longdesc lang="en">
-Cluster check user password
+Cluster check user password. Empty passwords are ignored unless
+the parameter "check_passwd_use_empty" is set to 1.
 </longdesc>
 <shortdesc lang="en">check password</shortdesc>
 <content type="string" default="" />
 </parameter>
 
+<parameter name="check_passwd_use_empty" unique="0" required="0">
+<longdesc lang="en">
+Use an empty "check_passwd" password. If this parameter is set to 1
+and "check_passwd" is an empty string, an empty password is used
+when calling the "mysql" client binary.
+</longdesc>
+<shortdesc lang="en">check password use empty</shortdesc>
+<content type="boolean" default="0"/>
```

Comment:
This should be assigned a `OCF_RESKEY_check_passwd_use_empty_default` value to be used for `default=` here, and assigned to `OCF_RESKEY_check_passwd_use_empty`:
https://github.com/ClusterLabs/resource-agents/blob/master/heartbeat/nagios#L19-L36

Reply:
Done